migrations/2_deploy_contracts.js

```diff
@@ -1,30 +1,36 @@
-const LinniaHub = artifacts.require("./LinniaHub.sol")
-const LinniaUsers = artifacts.require("./LinniaUsers.sol")
-const LinniaRecords = artifacts.require("./LinniaRecords.sol")
-const LinniaPermissions = artifacts.require("./LinniaPermissions.sol")
+const LinniaHub = artifacts.require("./LinniaHub.sol");
+const LinniaUsers = artifacts.require("./LinniaUsers.sol");
+const LinniaRecords = artifacts.require("./LinniaRecords.sol");
+const LinniaPolicies = artifacts.require("./LinniaPolicies.sol");
+const LinniaPermissions = artifacts.require("./LinniaPermissions.sol");
 
 module.exports = (deployer, network, accounts) => {
-  const adminAddress = accounts[0]
-  let hubInstance
+  const adminAddress = accounts[0];
+  let hubInstance;
   // deploy the hub
   deployer.deploy(LinniaHub).then(() => {
-    return LinniaHub.deployed()
+    return LinniaHub.deployed();
```

Comment:
Can adding the semi be part of another PR?  It makes it hard to see what changed.

Reply:
Can you highlight what actually has changed here?  The added semicolons have made the entire file show up in the diff.

Reply:
The new code shows up as an extra dense highlight. I just added the Policy contract migration to the script using the same pattern that we use for the other contrats

Reply:
+1

Reply:
ok